BurnerB/PropertyProLite-CH2
# #[167189170]Fix Api Responses

#### What does this PR do?
Fix response messages
#### Description of Task to be completed?
Response messages should occur in the required format
#### How should this be manually tested?
clone this repo
run `npm start`
visit any link on postman:
with valid input data the success response should be:
```
"status" : 'success',
"data" : {}
```
and on failure:
```
"status" : 'error',
"error" : "relevant-error-message"
```
#### What are the relevant pivotal tracker stories?
[#167189170](https://www.pivotaltracker.com/story/show/167189170)
#### Screenshots (if appropriate)
![RESPONSEEE](https://user-images.githubusercontent.com/45785786/61011477-68743400-a383-11e9-9b99-ace5a4ee49d8.png)
